examples/simple_polaris_client.cc

```diff
@@ -27,7 +27,7 @@ using namespace point_one::polaris;
 DEFINE_string(polaris_api_key, "",
               "The polaris API key. Sign up at app.pointonenav.com.");
 
-DEFINE_string(polaris_unique_id, "",
+DEFINE_string(polaris_unique_id, "device12345",
```

Comment:
Iirc, we removed this so the users using our client don't automatically all use the same device I'd and then make a lot of collisions for us.

Reply:
I changed this to match the C example, but I'll switch them the other way and return an error if a unique ID is not specified.